astropy/wcs/wcs.py

```diff
@@ -1898,7 +1898,7 @@ def all_world2pix(self, *args, **kwargs):
                HST's ACS/WFC detector, which has the strongest
                distortions of all HST instruments, testing has
                shown that enabling this option would lead to a about
-               50-100\% penalty in computational time (depending on
+               50-100\\% penalty in computational time (depending on
```

Comment:
why does this need a double `\\`? I thought the point here was to include just a `%` sign (which is not used for formatting). [note: am not sure any backslash is necessary at all.]